WeakAurasOptions/SubRegionOptions/LinearProgressTexture.lua

```diff
@@ -0,0 +1,268 @@
+    },
+    -- Doesn't appear to work
+    linearTextureRotation = {
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      name = L["Texture Rotation"],
+      desc = L["Uses Texture Coordinates to rotate the texture."],
+      order = 15,
+      min = 0,
+      max = 360,
+      bigStep = 1
+    },
+    -- Doesn't appear to work
+    linearTextureAuraRotation = {
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      name = L["Rotation"],
+      order = 16,
+      min = 0,
+      max = 360,
+      bigStep = 1
+    },
+
+    progress_source = {
+      type = "select",
+      width = WeakAuras.normalWidth,
+      name = L["Progress Source"],
+      order = 17,
+      control = "WeakAurasTwoColumnDropdown",
+      values = OptionsPrivate.Private.GetProgressSourcesForUi(parentData, true),
+      get = function(info)
+        return OptionsPrivate.Private.GetProgressValueConstant(data.progressSources or {-2, ""})
+      end,
+      set = function(info, value)
+        if value then
+          data.progressSources = data.progressSources or {}
+          -- Copy only trigger + property
+          data.progressSources[1] = value[1]
+          data.progressSources[2] = value[2]
+        else
+          data.progressSources = nil
+        end
+        WeakAuras.Add(parentData)
+      end,
+    },
+
+    progress_source_space = {
+      type = "description",
+      name = "",
+      order = 18,
+      width = WeakAuras.normalWidth,
+    },
+
+    anchor_mode = {
+      name = L["Anchor Mode"],
+      type = "select",
+      width = WeakAuras.normalWidth,
+      order = 19,
+      values = OptionsPrivate.Private.anchor_mode,
+    },
+
+    anchor_area = {
+      name = L["Area"],
+      type = "select",
+      width = WeakAuras.normalWidth,
+      control = "WeakAurasTwoColumnDropdown",
+      order = 20,
+      values = areaAnchors,
+      hidden = function()
+        return data.anchor_mode == "point"
+      end
+    },
+
+    self_point = {
+      name = L["Anchor"],
+      type = "select",
+      width = WeakAuras.normalWidth,
+      control = "WeakAurasTwoColumnDropdown",
+      order = 21,
+      values = OptionsPrivate.Private.point_types,
+      hidden = function()
+        return data.anchor_mode == "area"
+      end
+    },
+
+    anchor_point = {
+      name = L["To Region's"],
+      type = "select",
+      width = WeakAuras.normalWidth,
+      control = "WeakAurasTwoColumnDropdown",
+      order = 22,
+      values = pointAnchors,
+      hidden = function()
+        return data.anchor_mode == "area"
+      end
+    },
+
+    width = {
+      name = L["Width"],
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      min = 0,
+      softMax = 200,
+      step = 1,
+      order = 23,
+      hidden = function()
+        return data.anchor_mode == "area"
+      end
+    },
+
+    height = {
```

Comment:
model subelement may need an update to match textures/stopmotion subelements size & positioning options

Reply:
I did look at that briefly and I agree, but it seems this requires a bit of tinkering to get right, I'll do that some other day.

Reply:
I think there are 3 cases for sub element position settings:
Area + Point: (Linear/Circle) Texture, StopMotion, **Border!, Glow!, Model!**
Only Point anchoring: Text
Special: Ticks

There's not a lot of value in having Border and Glow being abled to anchor at a point with a specific size atm, but I think they are in same category as e.g. linear progres textures. I'll put changing those 3 on my todo-list but I don't think it's that important for now.

While I made Ticks being abled to provide anchors for other sub elements, and thus the new sub element types can anchor to ticks, I didn't made these new sub elements themselves provide anchors. That's because doing that requires preventing cycles in the anchoring, which are currently simply impossible.